### DISCUSSION TITLE
请问passwall与passwall2依赖的防火墙版本是fw4吗

### DISCUSSION THREAD

**Discussion by careysucci** [Openwrt-Passwall/openwrt-passwall2#680] [Q&A]
请问passwall与passwall2依赖的防火墙版本是fw4吗
如果不是fw4，可以怎么配置只保留fw4相关的依赖库及规则，但不影响功能

**Comment by zpvel** [comment]
我比较少用passwall ，刷到你这条， 回复你第一条。  高级设置里面-转发配置-防火墙工具。 可以选择nftables和iptables

**Reply by careysucci** [reply]
感谢答复
